packages/manager/modules/pci/src/projects/new/payment/controller.js

```diff
@@ -1,4 +1,15 @@
+import find from 'lodash/find';
+import {
+  ORDER_FOLLOW_UP_STATUS_ENUM,
+  ORDER_FOLLOW_UP_STEP_ENUM,
+} from '../../projects.constant';
+
 const getPaymentMethodTimeoutLimit = 30000;
+const ANTI_FRAUD = {
+  CASE_FRAUD_REFUSED: '(error 906)',
```

Comment:
Why do we have to hardcode this ?

Reply:
We used as pattern for case f for anti-fraud.
At checkout step we can't get the history billing info.

No other solutions, i discussed on it with PCI team.

Reply:
So if they decide to change the pattern this won't work anymore, we should rely on something more consistent like an Enum value or stuff like that

Reply:
Yes, i am aware about that @marie-j.
In this case the only solution that we found is to catch it from the error message.
For some security purposes, call to checkout don't allow to return more info (extra field).
So we decide to use the message to populate message with anti-fraud error code.

Thanks.

Reply:
How about the error code 906?

Reply:
With PCI team we don't found another solution except that.
For security purposes they are some restrictions from order (checkout).
We can't have more response fields except `class and message`, so we used message to catch F case.